docs/create-gh-validate-interceptor.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: create-github-validate-interceptor
+spec:
+  inputs:
+    params:
+      - name: Name
+        description: "The name of the interceptor Service."
+      - name: GitHubSecretName
+        description: "The Secret name for GitHub webhook secret. This is always mounted and must exist"
+      - name: GitHubSecretStringKey
+        description: "Key name within the Secret for the Github webhook secret."
+      - name: ContainerImage
+        description: "Container image of the Github Validation Interceptor"
+        default: "gcr.io/tekton-releases/github.com/tektoncd/triggers/cmd/gh-validate@sha256:98e4f1ac48fbad76f51d5eae68147c9800dc4c9a2e95a60532c92fc4c8d204b1"
+  steps:
+    - name: create-service
+      image: lachlanevenson/k8s-kubectl:latest
+      command:
+        - sh
+      args:
+        - -ce
+        - |
+          set -e
+          cat <<EOF | kubectl create -f -
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: $(inputs.params.Name)
+          spec:
+            type: ClusterIP
+            selector:
+              app: $(inputs.params.Name)
+            ports:
+              - protocol: TCP
+                port: 80
+                targetPort: 8080
+          EOF
+          fi
```

Comment:
I'm pretty sure the fi's cause errors (we don't have any ifs after all)

```
adams-mbp:triggers_prototype aroberts$ k logs github-validate-run-89mhl-pod-195926 --all-containers
{"level":"warn","ts":1570624241.7803333,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/master\" is not a valid GitHub commit ID"}
{"level":"info","ts":1570624241.781536,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}
service/my-interceptor-service created
sh: syntax error: unexpected "fi"
```

Same with the other